structuretoolkit/analyse/symmetry.py

```diff
@@ -357,9 +357,13 @@ def get_primitive_cell(
         new_structure = self._structure.copy()
         new_structure.cell = cell
         new_structure = new_structure[: len(indices)]
```

Comment:
The primary issue is this line, because the resulting new structure can have only a subset of the different kinds of elements the original structure had.

Reply:
Right, in the case `use_elements==False`. In the case `use_elements==True`, I presume we do indeed want all the same elements in the original structure. I honestly don't understand what the impact of all this is, I just want to point out that (as far as I can tell) the _default_ behaviour should be to have all the same elements.